config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :dishes, only: [:show] do
+    resources :dish_ingredients, only: [:update]
+  end
+
+  resources :chefs, only: [:show] do
+    resources :ingredients, module: :chef, only: [:index]
```

Comment:
Nice RESTful routing with advanced syntax here!

Remember that on an IC it's totally fine to handroll if you'd like!